README.md

```diff
@@ -1,6 +1,6 @@
 # xcrun
 
-This is a Python wrapper around the `xcrun` utility that Apple provides for interacting with the various Xcode developer tools. 
+This is a Python wrapper around the `xcrun` utility that Apple provides for interacting with the various Xcode developer tools. This module has been tested against python 2.7, 3.6 & 3.7.
```

Comment:
The tests may work against 2.7, but we don't support it. You should remove any references to 2.7.